api/ExtendableMessageEvent.json

```diff
@@ -69,14 +93,113 @@
             "edge_mobile": {
               "version_added": null
             },
-            "firefox": {
+            "firefox": [
+              {
+                "version_added": "45"
+              },
+              {
+                "version_added": "45",
+                "version_removed": "46",
+                "notes": "Service workers (and Push) have been disabled in the Extended Support Release"
+              },
+              {
+                "version_added": "52",
+                "version_removed": "53",
+                "notes": "Service workers (and Push) have been disabled in the Extended Support Release"
+              }
+            ],
+            "firefox_android": [
+              {
+                "version_added": "45"
+              },
+              {
+                "version_added": "45",
+                "version_removed": "46",
+                "notes": "Service workers (and Push) have been disabled in the Extended Support Release"
+              },
+              {
+                "version_added": "52",
+                "version_removed": "53",
+                "notes": "Service workers (and Push) have been disabled in the Extended Support Release"
+              }
+            ],
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
               "version_added": null
             },
-            "firefox_android": {
+            "opera_android": {
               "version_added": null
             },
-            "ie": {
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": {
+              "version_added": false
+            },
+            "samsunginternet_android": {
               "version_added": null
+            }
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "data": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/ExtendableMessageEvent/data",
+          "support": {
+            "webview_android": {
+              "version_added": false
+            },
+            "chrome": {
+              "version_added": "51"
+            },
+            "chrome_android": {
+              "version_added": "51"
+            },
+            "edge": {
+              "version_added": null
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": [
+              {
+                "version_added": "45"
+              },
+              {
+                "version_added": "45",
+                "version_removed": "46",
+                "notes": "Service workers (and Push) have been disabled in the Extended Support Release"
```

Comment:
So it was added again for non-ESR releases? The current way its written I'm not sure the MDN table parser will understand this.

Reply:
The table parser will understand, but there may well be a clearer way to write the note. Not sure what the full implications of those notes are.

Reply:
For now, we're treating Firefox ESR as a different browser (just like Firefox Nightly), so this shouldn't be represented with version_added and version_removed. In other data, we've just added a note (compare https://github.com/mdn/browser-compat-data/blob/master/api/Client.json), and I've updated this to also just use a note. We have https://github.com/mdn/browser-compat-data/issues/473 on file to find a better solution.